Tiltfile

```diff
@@ -2,6 +2,8 @@
 
 default_registry('gcr.io/windmill-public-containers')
 set_team('0584d8f6-05a2-49f5-923b-657afef098fe')
+username = str(local('whoami')).rstrip('\n')
+experimental_analytics_report({'user': username})
```

Comment:
There's an existing legacy tag named 'user', so we should probably avoid using that here.

How about 'user.name'?

Reply:
Sure. Changed.